tests/phpunit/elementor/app/import-export/processes/test-import.php

```diff
@@ -475,6 +475,49 @@ public function test_get_manifest__kit_library() {
 		}
 	}
 
+	public function test_init_import_session__kit_thumbnail() {
```

Comment:
maybe something like this? to use the same convention as the previous tests:

```suggestion
	public function test_run__get_kit_thumbnail_from_manifest() {
```

```suggestion
	public function test_run__get_kit_thumbnail_from_api() {
```

Reply:
where are the changes?